domain/src/main/java/com/nadosunbae_android/domain/model/user/UserPostData.kt

```diff
@@ -0,0 +1,25 @@
+package com.nadosunbae_android.domain.model.user
+
+import java.util.*
+
+data class UserPostData(
+    val commentCount: Int,
+    val content: String,
+    val createdAt: Date,
+    val like: Like,
+    val majorName: String,
+    val postId: Int,
+    val title: String,
+    val type: String?,
+    val writer: Writer
+) {
+    data class Like(
+        val isLiked: Boolean,
+        val likeCount: Int
+    )
+
+    data class Writer(
+        val id: Int,
+        val nickname: String
+    )
```

Comment:
```suggestion
data class UserPostData(
    val commentCount: Int,
    val content: String,
    val createdAt: Date,
    val isLiked: Boolean,
    val likeCount: Int
    val majorName: String,
    val postId: Int,
    val title: String,
    val type: String?,
     val writerId: Int,
     val nickname: String
) 
```
이렇게 줄여서 쓸 수도 있을 것 같아요!

Reply:
옹 감사합니다!